app/response.py

```diff
@@ -8,10 +9,19 @@ def success(values, message):
 
     return make_response(jsonify(res)), 200
 
+
 def badRequest(values, message):
     res = {
         'data': values,
         'message': message
     }
 
-    return make_response(jsonify(res)), 400
\ No newline at end of file
+    return make_response(jsonify(res)), 400
+
+
+def internal_server_error(message="Sorry, seems like your input is not in right format"):
```

Comment:
The default message can be simpler:
`Incorrect format`

Reply:
done